plugins/catalog-backend/schema/openapi.ts

```diff
@@ -0,0 +1,1320 @@
+/*
```

Comment:
@sennyeya Ping for this question. This file won't be shipped with the package since it's not in the `files` section so it will NOT be present at runtime once the package is published to npm.

Reply:
Sorry, lost this conversation in the review. It makes sense to include the yaml file in the exports, but the ts seems like it should just be a compile time dependency for internal development. There's no harm in adding it, but it's going to be large especially for bigger plugins. It looks like `catalog-model` uses the `src` folder, so I think that would make sense to use here as well.